src/common/Tooltip/tooltip.scss

```diff
@@ -4,4 +4,6 @@
   font-size: 15px;
   line-height: 18px;
   white-space: nowrap;
+  display: flex;
+  max-width: 400px;
```

Comment:
These styles broke the tooltip. Why do you need it?

Reply:
because when there is a lot of text the line doesn't break and gets off screen